src/systems/a320_systems/src/hydraulic/mod.rs

```diff
@@ -1218,18 +1218,16 @@ impl A320Hydraulic {
     const HIGH_PITCH_PTU_SOUND_DURATION: Duration = Duration::from_millis(3000);
 
     const FLAP_FFPU_TO_SURFACE_ANGLE_BREAKPTS: [f64; 12] = [
-        0., 65., 115., 120.53, 136., 145.5, 152., 165., 168.3, 179., 231.2, 251.97,
-    ];
-    const FLAP_FFPU_TO_SURFACE_ANGLE_DEGREES: [f64; 12] = [
-        0., 10.318, 18.2561, 19.134, 21.59, 23.098, 24.13, 26.196, 26.72, 28.42, 36.703, 40.,
+        0., 35.66, 69.32, 89.7, 105.29, 120.22, 145.51, 168.35, 189.87, 210.69, 231.25, 251.97,
     ];
+    const FLAP_FFPU_TO_SURFACE_ANGLE_DEGREES: [f64; 12] =
```

Comment:
I think there might've been a typo here in the original code - surely this refers to the F**P**PU?

Reply:
Oh yeah, it should be FPPU. I'll just correct it in this PR